app/components/Footer/footer.css

```diff
@@ -10,3 +10,11 @@
   align-items: baseline;
   margin: var(--spacing-104) 13.33vw var(--spacing-80) 13.33vw;
 }
+
+@media (max-width: 600px) {
+  .footer-contents {
+    flex-direction: column;
+    gap: var(--spacing-32);
```

Comment:
I think using spacing-80 var, making is 40px for mobile. Sorry for my not super organized system, will try to make it better soon.